Sheepish-Squirrels/bangazon-api-sprint1
# Krorderproduct

Fixes #8

Changes proposed in this pull request:

created file to generate order products table storing into JSON file

edited orders model to get one order with product details and to get all orders with product details

Steps to test:

In terminal,
run nodemon app.js
run db/build-data.js to ensure faker json data is being created
run db/build-db.js to ensure sql table is being created
Make sure you can GET all orders with product details, GET one order with product details
